11/hmw-11-01.py

```diff
@@ -0,0 +1,20 @@
+# Написати власний декоратор, задачею якого має бути друк назви функції і часу, коли вона була викликана.
+# Декоратор має працювати для різних функцій однаково.
+
+from datetime import datetime
+
+
+def my_decorator(func):
+    def deco_func():
+        print(f'Function name is "{func.__name__}" and the time of call is {datetime.now()}')
+        func()
```

Comment:
Add here `return` to be able to decorate functions that return values. And pass here *args and **kwargs according to my previous comment